lib/msal-common/src/uCache/serialize/Serializable.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+export class Serializable {
+    toObject(obj: any) {
```

Comment:
This would potentially benefit from generics, which would preferably over `any`.

Reply:
yep. And I believe will also help with the above file. Will make the changes.

Reply:
I had couple of implementations to arrive here, I now think it makes sense to make this a utility instead of a base class and works well with generics. Checked in the modified code.

Reply:
Got rid of the base class and added a helper.